### PR TITLE
Remove nucleation nested parameter struct

### PR DESCRIPTION
**Pull request by nefrathenrici**
Content
- Remove nucleation nested parameter struct, place in top-level
- Extend the new parameter scheme (0 moment, P3) to the nucleation parameters

### CODE REVIEW

**Comment by trontrytel** [comment]
This should be a breaking change? Should we bump the version to 0.14?

**Comment by nefrathenrici** [comment]
> This should be a breaking change? Should we bump the version to 0.14?

@trontrytel This will be a breaking change, I will up the version when the PR is ready

File: src/parameters/modal_nucleation.jl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    param_set.a_4,
+    param_set.a_5,
+    param_set.Y_MTO3,
+    param_set.Y_MTOH,
+    param_set.k_MTO3,
+    param_set.k_MTOH,
+    param_set.exp_MTO3,
+    param_set.exp_MTOH,
+    param_set.k_H2SO4org,
+)
```
Comment (charleskawczynski) [inline src/parameters/modal_nucleation.jl:78]:
```suggestion
function ModalNucleationParameters(param_set)
    fns = fieldnames(ModalNucleationParameters)
    vals = getproperty.(Ref(param_set), fns)
    nt = (;zip(fns, vals)...)
    ModalNucleationParameters(;nt...)
end
```
I think this should work (safely) too, if all of the names are consistently stored in `param_set`

**Comment by trontrytel** [comment]
The docs error should be fixed now on main

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "CloudMicrophysics"
 uuid = "6a9e3e04-43cd-43ba-94b9-e8782df3c71b"
 authors = ["Climate Modeling Alliance"]
-version = "0.13.3"
+version = "0.14.0"
```
Comment (trontrytel) [inline Project.toml:4]: Could I ask you to update ClimaAtmos and Kinematic1D models to the new release?

**Review by trontrytel — APPROVED** [review]
Thank you!

**Pull request merged**